client/src/featureform/serving.py

```diff
@@ -236,6 +241,10 @@ def nearest(self, name, variant, vector, k):
         resp = self._stub.Nearest(req)
         return resp.entities
 
+    def parse_proto_value(self, value):
```

Comment:
this can be static, in fact it should probz be part of some proto utils file

Reply:
Done